tests/framework/installer/ceph_manifests.go

```diff
@@ -572,7 +572,9 @@ func (m *CephManifestsMaster) GetBlockPoolStorageClass(namespace string, poolNam
 	return concatYaml(m.GetBlockPoolDef(poolName, namespace, "1"), m.GetBlockStorageClassDef(poolName, storageClassName, reclaimPolicy, namespace, false))
 }
 
-func (m *CephManifestsMaster) GetFilesystem(namespace, name string) string {
+// GetFilesystem returns the manifest to create a Rook filesystem resource with the given config.
+func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount int32) string {
+	ac := fmt.Sprintf("%d", activeCount)
```

Comment:
For the int conversion you could use `strconv.Itoa(activeCount)`. Including it inline below works well too in these yamls.